systemtests/src/main/java/io/enmasse/systemtest/utils/IoTUtils.java

```diff
@@ -58,6 +59,17 @@ public static void waitForIoTConfigReady(Kubernetes kubernetes, IoTConfig config
         TestUtils.waitForNReplicas(kubernetes, EXPECTED_DEPLOYMENTS.length, IOT_LABELS, budget);
     }
 
+    public static void deleteIoTConfigAndWait(Kubernetes kubernetes, IoTConfig config) throws Exception{
+        String operationID = TimeMeasuringSystem.startOperation(SystemtestsOperation.DELETE_IOT_CONFIG);
+        kubernetes.getIoTConfigClient().withName(config.getMetadata().getName()).delete();
+        waitForIoTConfigDeleted(kubernetes, config);
+        TimeMeasuringSystem.stopOperation(operationID);
+    }
+
+    private static void waitForIoTConfigDeleted(Kubernetes kubernetes, IoTConfig config) throws Exception{
```

Comment:
why passing `IoTConfig config` if it's not used?

Reply:
my mistake, but this doesn't affect to the problem

Reply:
I guess so, I was just wondering :)